<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainMenu</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QLineEdit" name="txtFolderPath">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>C:/JP</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3" rowspan="3">
     <widget class="QTabWidget" name="twFolderMetadata">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabFolderData">
       <attribute name="title">
        <string>Folder Info</string>
       </attribute>
       <widget class="QPushButton" name="btnSaveData">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>510</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtName">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>340</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>27</x>
          <y>20</y>
          <width>30</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Name: </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>20</y>
          <width>40</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Score: </string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spbScore">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>20</y>
          <width>40</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
       <widget class="QTextEdit" name="txtComment">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>260</y>
          <width>341</width>
          <height>140</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Comment: </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>20</y>
          <width>40</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Image:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblImage">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>50</y>
          <width>240</width>
          <height>330</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtImageUrl">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>410</y>
          <width>240</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>390</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Change Image URL: </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtWebUrl">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>410</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>410</y>
          <width>40</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Web Url: </string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnViewUrl">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>410</y>
          <width>50</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>View site</string>
        </property>
       </widget>
       <widget class="QListWidget" name="lwGenres">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>300</width>
          <height>160</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>40</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Genres:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnAddGenre">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>100</y>
          <width>30</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnRemoveGenre">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>130</y>
          <width>30</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cmbGenre">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>301</width>
          <height>22</height>
         </rect>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Action</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Adventure</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cars</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Comedy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dementia</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Demons</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Drama</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ecchi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fantasy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Game</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Harem</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hentai</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Historical</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Horror</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Josei</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Kids</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Magic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Martial Arts</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mecha</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Military</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Music</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mystery</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Parody</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Police</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Psychological</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Romance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Samurai</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>School</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sci-Fi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Seinen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shoujo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shoujo Ai</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shounen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shounen Ai</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Slice of Life</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Space</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sports</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Super Power</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Super Natural</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thriller</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vampire</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yaoi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yuri</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="tabRootExplorer">
       <attribute name="title">
        <string>Filter Selected Directory</string>
       </attribute>
       <widget class="QPushButton" name="btnRemoveGenreFilter">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>90</y>
          <width>30</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cmbGenreFilter">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>301</width>
          <height>22</height>
         </rect>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Action</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Adventure</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cars</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Comedy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dementia</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Demons</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Drama</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ecchi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fantasy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Game</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Harem</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hentai</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Historical</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Horror</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Josei</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Kids</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Magic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Martial Arts</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mecha</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Military</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Music</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mystery</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Parody</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Police</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Psychological</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Romance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Samurai</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>School</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sci-Fi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Seinen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shoujo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shoujo Ai</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shounen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shounen Ai</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Slice of Life</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Space</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sports</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Super Power</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Super Natural</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thriller</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vampire</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yaoi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yuri</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="btnAddGenreFilter">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>60</y>
          <width>30</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QListWidget" name="lwResults">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>270</y>
          <width>300</width>
          <height>270</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="lwGenresFilter">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>300</width>
          <height>160</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>40</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Genres:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>250</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Results:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnShowResults">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>210</y>
          <width>158</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Search Genres</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabFolderURL">
       <attribute name="title">
        <string>Reference URL (Web Browser)</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QWebView" name="wvMetadataURL">
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btnShowFolderContent">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Show Content</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QTreeView" name="tvFolders">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnBrowseForFolder">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
